### PR TITLE
LATTICE-1624 fix toputilizers query for linking entity sets

### PR DESCRIPTION
**Pull request by you-leee**

### CODE REVIEW

File: src/main/kotlin/com/openlattice/graph/Graph.kt [hunk 1]
```diff
@@ -297,12 +300,52 @@ class Graph(private val hds: HikariDataSource, private val edm: EdmManager) : Gr
...
+                    authorizedFilteredRanking.filteredRanking.associationAggregations,
+                    ASSOC
+            ).map { it.value to authorizedFilteredRanking.associationPropertyTypes[it.key]!!.datatype } +
+                    buildAggregationColumnMap(
+                            index,
+                            authorizedFilteredRanking.entitySetPropertyTypes,
+                            authorizedFilteredRanking.filteredRanking.neighborTypeAggregations,
+                            ENTITY
+                    ).map { it.value to authorizedFilteredRanking.entitySetPropertyTypes[it.key]!!.datatype } +
+                    (associationCountColumnName(index) to EdmPrimitiveTypeKind.Int64 )+
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/graph/Graph.kt]: What's up with the weird whitespace?

File: src/main/kotlin/com/openlattice/graph/Graph.kt [hunk 2]
```diff
@@ -356,8 +382,16 @@ class Graph(private val hds: HikariDataSource, private val edm: EdmManager) : Gr
                 filteredRankings.mapIndexed { index, authorizedFilteredRanking ->
                     val tableSql = buildAssociationTable(index, entitySetIds, authorizedFilteredRanking, linked)
                     //We are guaranteed at least one association for a valid top utilizers request
+
                     if (index == 0) {
-                        "SELECT *,$scoreColumn FROM ($tableSql) as assoc_table$index "
+                        val selectedColumns =
+                                if( linked ) {
+                                    " *, UUID('${linkingEntitySetId.get()}') as $SELF_ENTITY_SET_ID, ${LINKING_ID.name} as $SELF_ENTITY_KEY_ID, $scoreColumn "
```
Comment (geekbeast) [inline src/main/kotlin/com/openlattice/graph/Graph.kt]: Shouldn't you alias linking id to SELF_ENTITY_KEY_ID?

Comment (you-leee) [reply src/main/kotlin/com/openlattice/graph/Graph.kt]: It is: ${LINKING_ID.name} as $SELF_ENTITY_KEY_ID

File: src/main/kotlin/com/openlattice/graph/Graph.kt (hunk 1 above)
Comment (you-leee) [reply src/main/kotlin/com/openlattice/graph/Graph.kt]: What do you mean by that?

File: src/main/kotlin/com/openlattice/graph/Graph.kt (hunk 2 above)
Comment (geekbeast) [reply src/main/kotlin/com/openlattice/graph/Graph.kt]: Ah, for some reason it got cut off by github. LGTM.

File: src/main/kotlin/com/openlattice/graph/Graph.kt (hunk 1 above)
Comment (geekbeast) [reply src/main/kotlin/com/openlattice/graph/Graph.kt]: There is a space between Int64 and paranthesis but not so on the next line. There is also no space before the plus, but other plus signs have a space.

**Pull request merged**